x-pack/metricbeat/Jenkinsfile.yml

```diff
@@ -115,14 +115,14 @@ stages:
                 - "windows-7"
             branches: true     ## for all the branches
             tags: true         ## for all the tags
-    windows-7-32:
-        mage: "mage build unitTest"
-        platforms:             ## override default labels in this specific stage.
-            - "windows-7-32-bit"
-        when:                  ## Override the top-level when.
-            comments:
-                - "/test x-pack/metricbeat for windows-7-32"
-            labels:
-                - "windows-7-32"
-            branches: true     ## for all the branches
-            tags: true         ## for all the tags
+#    windows-7-32:
+#        mage: "mage build unitTest"
+#        platforms:             ## override default labels in this specific stage.
+#            - "windows-7-32-bit"
+#        when:                  ## Override the top-level when.
+#            comments:
+#                - "/test x-pack/metricbeat for windows-7-32"
+#            labels:
+#                - "windows-7-32"
+#            branches: true     ## for all the branches
+#            tags: true         ## for all the tags
```

Comment:
We're disabling metricbeat windows7 32bit testing for now because the linker gets OOM when trying to build metricbeat (which has many dependencies).

Please open a follow up issue. We are loosing many tests here and need to see what we can do to resurrect those.

Reply:
Done.